### PR TITLE
Client: Sequence diagram for UpdateRepoHookHandler

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: systems/WikibaseClient/06-Runtime_View.md [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+    opt Not connected to Items
+        UpdateRepo-HookHandler-->>MediaWiki Page: Return control flow
+    end
+    UpdateRepo-HookHandler->>+Repo MediaWiki JobQueue: Notify Repo of the changes (job)
+    opt Repo Notified
+        UpdateRepo-HookHandler-->>MediaWiki Page: Return control flow
+    end
+```
+
+## Data Bridge
```
Comment (tzhelyazkova) [inline systems/WikibaseClient/06-Runtime_View.md:23]: This will create merge conflicts with the data bridge sequence diagram PR

Comment (addshore) [reply systems/WikibaseClient/06-Runtime_View.md]: Fixed!

**Review by jakobw — APPROVED** [review]
I like it!

At first I was thinking that it would be more useful to know how the Repo reacts to this kind of notification with a sitelink change, but it would probably be better to document that half on the Repo side, and then link back and forth between the two.

**Comment by addshore** [comment]
> I like it!
> 
> At first I was thinking that it would be more useful to know how the Repo reacts to this kind of notification with a sitelink change, but it would probably be better to document that half on the Repo side, and then link back and forth between the two.

Indeed, when I first started the diagram I ended up including more repo detail, then remembered that I was meant to be documenting client.

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**